HowTos/SAML_Integration_PingOne_IdP.rst

```diff
@@ -0,0 +1,259 @@
+   | Enforce signed Authn request | Disabled                                          |
+   +------------------------------+---------------------------------------------------+
+   | Verification certificate     | No Verification Certificates Selected             |
+   +------------------------------+---------------------------------------------------+
+
+   .. note::
+   
+      ``[host]`` is the hostname or IP of your Aviatrix controller.  For example, ``https://controller.demo.aviatrix.live``
+
+      ``[Endpoint Name]`` is an arbitrary identifier.  This same value should be used when configuring SAML in the Aviatrix controller.
+      
+      ``[Entity ID]`` is using ``https://[host]/`` as default if you select `Hostname` option when configuring SAML in the Aviatrix controller.
+      
+   |pingone_idp_configuring_saml_connection|
+      
+#. Click Save and Continue.
+
+#. For attribute mapping, click the button "+ADD ATTRIBUTE" and then select "PingOne Attribute" to map PingOne user attribute to an attribute in this application as below.
+
+   +------------------------+-----------------------+
+   | PINGONE USER ATTRIBUTE | APPLICATION ATTRIBUTE |
+   +------------------------+-----------------------+
+   | User ID                | saml_subject          |
+   +------------------------+-----------------------+
+   | Given Name             | FirstName             |
+   +------------------------+-----------------------+
+   | Family Name            | LastName              |
+   +------------------------+-----------------------+
+   | Email Address          | Email                 |
+   +------------------------+-----------------------+
+   
+   .. note::
+   
+      Notes: User ID is a default required in PingOne
+
+   ||pingone_idp_configuring_attribute_mapping||
+
+#. Click Save and Close.
+
+#. Enable the WEB SAML APP
+
+   |pingone_idp_enable|
+   
+.. _pingone_idp_metadata:
+
+Step 3. Retrieve PingOne IdP metadata
+#####################################
+
+.. note::
+
+   This step is usually completed by the PingOne for Customers admin.
+
+#. After the application is created in PingOne, click Connections on the top of the page and then click Applications on the left.
+
+#. Locate the Web SAML application that we just created.
+
+#. Click the details icon to expand the Web SAML application and then click the button "Configuration". 
+
+#. Copy the URL from the IDP Metadata URL from the CONNECTION DETAILS. This value will be used to configure the Aviatrix SP Endpoint.
+
+  |pingone_idp_retrieve_idp_metadata_url|
+
+.. _pingone_update_saml_endpoint:
+
+Step 4. Update Aviatrix SP Endpoint
+###################################
+
+.. note::   
+   This step is usually completed by the Aviatrix admin. PineOne IdP provides IdP Metadata through URL obtained in Retrieve `PingOne IdP metadata URL <#pingone-idp-metadata>`__ step. PingOne for Customers IdP requires a custom SAML request template.
```

Comment:
Is the  "custom SAML request template." the one provided below in (.. code-block:: XML) line 202-209.
Is it the same one used for both controller and OpenVPN : Custom SAML Request Template?

Reply:
Yes, that is the same one used for both controller and OpenVPN